module/VuFind/src/VuFindTest/Integration/MinkTestCase.php

```diff
@@ -731,14 +737,21 @@ protected function logWarning(string $consoleMsg, string $logMsg = ''): void
      */
     public function setUp(): void
     {
-        // Give up if we're not running in CI:
+        // Give up if we're not running in CI (throws, so no problem with any
+        // further actions in any setUp methods of child classes):
         if (!$this->continuousIntegrationRunning()) {
             $this->markTestSkipped('Continuous integration not running.');
             return;
         }
 
         // Reset the modified configs list.
         $this->modifiedConfigs = [];
+
+        // Create a pathResolver:
+        $localDirs = strlen(trim(LOCAL_OVERRIDE_DIR)) > 0
```

Comment:
... and this, too, though I guess in this context you'd have to pass it a dummy container to satisfy the interface.